packages/hub/src/components/layout/RootLayout/PageFooter.tsx

```diff
@@ -1,16 +1,73 @@
 "use client";
 import { FC } from "react";
+import { FaGithub, FaDiscord, FaRss } from "react-icons/fa";
+import { SiGraphql } from "react-icons/si";
+import Image from "next/image";
+import logoPic from "@/public/logo.png";
+import Link from "next/link";
+import {
+  DISCORD_URL,
+  GITHUB_URL,
+  NEWSLETTER_URL,
+  QURI_DONATE_URL,
+} from "@/lib/common";
+import { aboutRoute, privacyPolicyRoute, termsOfServiceRoute } from "@/routes";
+
+const linkClasses = "items-center flex hover:text-gray-900";
 
 export const PageFooter: FC = () => (
-  <div className="p-8 border-t border-t-slate-200 bg-slate-100">
-    <div className="text-sm text-slate-500">
-      By{" "}
-      <a
-        href="https://quantifieduncertainty.org"
-        className="text-blue-500 hover:underline"
-      >
-        QURI
-      </a>
+  <div className="px-8 py-4 border-t border-t-slate-200 bg-slate-100">
+    <div className="text-sm text-slate-400 flex flex-column justify-between max-w-4xl mx-auto">
+      <div className="flex items-center gap-x-5 h-16">
```

Comment:
I'm not sure why `h-16` here is necessary.

Reply:
I wanted to keep this logo roughly on the top. Without this, it was horizontally centered, which seemed worse.